Begin.py/begin.py

```diff
@@ -52,76 +49,85 @@ def __repr__(self):
         s += 'MIJN EIGENSCHAP:\n' + str(self.my_feature)
         return s
 
-
     # Voeg hier andere methodes toe.
     # Je hebt in het bijzonder methodes nodig die het model vullen.
 
     # METHODS
-    def read_text_from_file(self, filename):
+
+    @staticmethod
+    def read_text_from_file(filename):
         """
-        methode:    read_text_from_file(self, filename) 
+        methode:    read_text_from_file(self, filename)
         argument:   self; verwijst naar de klasse TextModel en maakt hiervan 'a instance of a class'
-        argument:   filename; van het type string. 
-                    self.text: variabele waaraan de inhoud toegekend is 
-                    van het bestand filename. Type: 1 hele lange string.   
+        argument:   filename; van het type string.
+                    self.text: variabele waaraan de inhoud toegekend is
+                    van het bestand filename. Type: 1 hele lange string.
         """
 
-        with open("/home/annemarleen/programming/CS_for_all/Eindopdracht/test.txt") as f:
-            self.text = f.read().replace("\n", "").rstrip("")            
```

Comment:
op basis van de parameters leek make_sentence_lengths() de text via parameter te willen. Hierom heb ik self.text weggehaald omdat die niet gebruikt zou worden in dat geval.